src/lib/schema/feature-schema.ts

```diff
@@ -159,4 +159,5 @@ export const featureTagSchema = joi.object().keys({
     tagValue: joi.string(),
     type: nameType.optional(),
     value: joi.string().optional(),
+    createdByUserId: joi.number().optional(),
```

Comment:
I think this is used to validate input. And I believe all new input should have a createdById (but maybe it's a breaking change for the API)